### PR TITLE
Passive entity mappings

### PR DESCRIPTION
**Pull request by liach**
Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

File: mappings/net/minecraft/entity/ai/brain/Brain.mapping [hunk 1]
```diff
@@ -65,6 +66,8 @@ CLASS net/minecraft/class_4095 net/minecraft/entity/ai/brain/Brain
 		ARG 1 type
 		ARG 2 value
 		ARG 3 startTime
+	METHOD method_24526 resetActivities (Lnet/minecraft/class_4168;)V
```
Comment (YanisBft) [inline mappings/net/minecraft/entity/ai/brain/Brain.mapping:69]: Singular then?

File: mappings/net/minecraft/entity/passive/ParrotEntity.mapping [hunk 2]
```diff
@@ -1,24 +1,33 @@
 CLASS net/minecraft/class_1453 net/minecraft/entity/passive/ParrotEntity
+	FIELD field_6818 animationRadian F
+	FIELD field_6819 AnimationPercentage F
```
Comment (YanisBft) [inline mappings/net/minecraft/entity/passive/ParrotEntity.mapping:3]:
```suggestion
	FIELD field_6819 animationPercentage F
```

File: mappings/net/minecraft/entity/passive/RabbitEntity.mapping [hunk 3]
```diff
@@ -39,10 +50,15 @@ CLASS net/minecraft/class_1463 net/minecraft/entity/passive/RabbitEntity
 			ARG 1 type
 	CLASS class_1467 RabbitJumpControl
 		FIELD field_6855 rabbit Lnet/minecraft/class_1463;
+		FIELD field_6856 inJump Z
```
Comment (YanisBft) [inline mappings/net/minecraft/entity/passive/RabbitEntity.mapping:53]: `jumping`?

Comment (liach) [reply mappings/net/minecraft/entity/passive/RabbitEntity.mapping:53]: this is like in air... this is really just set, not read in any way.

**Comment by liach** [comment]
Closing in favor of #1231

**Pull request closed without merging**